schemas/categories.txt

```diff
@@ -0,0 +1,67 @@
+- Barrel Scanner
+A fixture which has a rotating cylinder of mirrors to reflect the light in a moving pattern.
+Example: american-dj/inno-pocket-fusion = https://www.adj.com/inno-pocket-fusion
+
+- Blinder
+Extremely bright fixtures intended to light the audience, rather than the stage.
+Example: elation/cuepix-blinder-ww2 = https://www.elationlighting.com/cuepix-blinder-ww2
+
+- Color Changer
+A fixture which can change its color electronically, especially (but not only) one with a gel wheel.  This includes RGB LEDs.
+Example: clay-paky/sharpy = https://www.claypaky.it/en/products/sharpy
+
+- Dimmer
+A device which takes mains power and DMX, and outputs PWM dimmed power, like for a conventional fixture to use.
+Example: chauvet-dj/dmx-4 = https://www.chauvetdj.com/products/dmx-4/
+
+- Effect
+A non-lighting special effect, such as pyro.
+Example: magicfx/stage-flame = https://www.magicfx.eu/en/home
+
+- Fan
+Large fans which also have LEDs on the fan blades.
+Example: glp/force-120 = https://www.glp.de/en/products/stage-effects/force-120
+
+- Flower
+(a.k.a. "moonflower")  A single fixture which produces many beams in different directions and often different colors
+Example: american-dj/revo-burst = https://www.adj.com/revo-burst
+
+- Hazer
+A thinner, more uniform and longer-lasting atmospheric effect than smoke/fog.
+Example: ultratec/radiance-hazer = https://ultratecfx.com/atmospherics/products-catalog/radiance-hazer/
+
+- Laser
+A fixture which uses lasers, often for the appearance of the beam in haze.
+Example: laserworld/cs-1000rgb = https://www.laserworld.com/en/laserworld-cs/laserworld-cs-1000rgb.html
+
+- Matrix
+A rectangular grid of lights, addressable either individually or by region, intended for the audience to see the pixels.
+Example: prolights/pixpan16 = https://www.prolights.it/product/PIXPAN16
+
+- Moving Head
+A fixture which can focus its beam by controlling the pan/tilt of the light itself.
+Example: martin/mac-aura = https://www.martin.com/en/products/mac-aura
+
+- Pixel Bar
+A linear array of lights, addressable either individually or by region, like for lighting a cyc.  The bar sometimes also has tilt (but not pan) control.
+Example: chroma-q/color-force-ii-72 = https://chroma-q.com/products/color-force-ii-72
```

Comment:
Please use "cyclorama" instead of "cyc"; "cyc" is hard to google.

Please also mention that sometimes the individual pixels are intended for the audience to see.